pep-0480.txt

```diff
@@ -236,56 +258,63 @@ End-to-End Signing
 ==================
 
 End-to-end signing allows both PyPI and developers to sign for the metadata
-downloaded by clients.  PyPI is trusted to make uploaded projects available to
-clients (PyPI signs the metadata for this part of the process), and developers
-sign the distributions that they upload to PyPI.
-
-In order to delegate trust to a project, developers are required to submit a
-public key to PyPI.  PyPI takes the project's public key and adds it to parent
-metadata that PyPI then signs.  After the initial trust is established,
-developers are required to sign distributions that they upload to PyPI using
-the public key's corresponding private key.  The signed TUF metadata that
-developers upload to PyPI includes information like the distribution's file
+downloaded by clients. PyPI is trusted to make uploaded projects available to
+clients (PyPI signs for the metadata for this part of the process), and
+developers sign for the distributions that they upload to PyPI.
+
+In order to delegate trust to a project, developers are required to submit an
+OpenID Connect identity or a public key to PyPI. PyPI takes the project's OIDC
```

Comment:
Forgive my ignorance, but can you say more about how PyPI would delegate from a targets role using an OIDC identity...

Reply:
I agree, some more detail here would be helpful. I think we could paraphrase a bit from https://docs.sigstore.dev/cosign/openid_signing/ a bit, and describe both the OAuth flow, and the identity token flow in an automated environment.